interface/modules/zend_modules/module/Acl/view/layout/layout.phtml

```diff
@@ -41,8 +41,7 @@ echo $this->doctype();
         <!-- Scripts -->
         <?php echo $this->headScript()
 					->prependFile($this->basePath() . '/js/application/common.js')
-          ->prependFile($this->basePath() . '/js/lib/html5.js', 'text/javascript', array('conditional' => 'lt IE 9',))
-					->prependFile($this->basePath() . '/js/lib/bootstrap.min.js')
+                    ->prependFile($this->basePath() . '/js/lib/bootstrap.min.js')
 					->prependFile($this->basePath() . '/js/acl/acl.js')
```

Comment:
There's going to be a lot of pain involved in updating the zend to modern js since they all use bunch of legacy js crutches for functionality. Rec doing this in a future PR (it is on my list of things to do at some point).